### PR TITLE
done navigation

### PR DESCRIPTION
**Pull request by chris-to-fer**
session: react nesting
challenge navigation

### CODE REVIEW

File: javascript/react-nesting_navigation/src/App.js [hunk 1]
```diff
@@ -1,34 +1,28 @@
 import "./styles.css";
+console.clear();
 
 import avatar from "./img/avatar.jpg";
```
Comment (itsTHL) [inline javascript/react-nesting_navigation/src/App.js:4]: You could remove this line and the following one as you don't use it in this file anymore, right?

**Comment by chris-to-fer** [comment]
Right I could remove it :)Von unterwegs gesendet...Am 16.01.2024 um 14:47 schrieb Tina Huynh Le ***@***.***>:﻿
@itsTHL commented on this pull request.



In javascript/react-nesting_navigation/src/App.js:
> @@ -1,34 +1,28 @@
 import "./styles.css";
+console.clear();
 
 import avatar from "./img/avatar.jpg";

You could remove this line and the following one as you don't use it in this file anymore, right?

**Comment by chris-to-fer** [comment]
Oh wait no I think  I need it I get it from hereVon unterwegs gesendet...Am 16.01.2024 um 14:47 schrieb Tina Huynh Le ***@***.***>:﻿
@itsTHL commented on this pull request.



In javascript/react-nesting_navigation/src/App.js:
> @@ -1,34 +1,28 @@
 import "./styles.css";
+console.clear();
 
 import avatar from "./img/avatar.jpg";

You could remove this line and the following one as you don't use it in this file anymore, right?

**Review by itsTHL — COMMENTED** [review]
Approved, but if you're up for a picky comment: You could follow Sven's recommendation to keep files as short and empty as possible, so refactor a bit as we discussed in the morning. Or you just try to do that in the future. :D

**Pull request merged**